### ISSUE TITLE
Allow form to be presented before calendar

### ISSUE THREAD

**Issue by Routhinator** [timekit-io/booking-js#128]
We'd like to force users to fill out the form before they are able to pick a timeslot, but it does not seem to be an option with Booking.js currently.

Have I missed something? Or can we mark this as a feature request?

**Comment by laander** [comment]
@Routhinator You're correct, it's not currently supported unfortunately. I've added the feature request label to this post :)

**Comment by karltaylor** [comment]
@Routhinator what's your build stack? You could just show a form and then show/hide the booking.js form when the form was submitted?
